src/workspaces/common/state/useWorkspaceDetails.tsx

```diff
@@ -4,9 +4,12 @@ import { Ajax } from 'src/libs/ajax';
 import { withErrorReporting } from 'src/libs/error';
 import { useCancellation, useOnMount } from 'src/libs/react-utils';
 import * as Utils from 'src/libs/utils';
```

Comment:
This file doesn't need to be TSX (it is just a hook function, not rendering TSX).